e2e/e2e.go

```diff
@@ -1,3 +1,18 @@
+/*
+Copyright 2020 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
```

Comment:
I can't recall details, but I remember there was some issue if there's no extra empty line after the license header... So if you look at gravity/etc. Go files, you'll notice there's an empty line b/w the license header and package declaration, I would do the same here.

```go
/*
Copyright 2018 Gravitational, Inc.

...
*/

package cli
...
```

Reply:
Fixed in ad01805.